pkg/config/con.go

```
@@ -0,0 +1,947 @@
+		return err
+	}
+
+	if len(stackConfigFilesAbsolutePaths) < 1 {
+		j, err := u.ConvertToYAML(cl.atmosConfig.IncludeStackAbsolutePaths)
+		if err != nil {
+			return err
+		}
+		errorMessage := fmt.Sprintf("\nno stack manifests found in the provided "+
+			"paths:\n%s\n\nCheck if `base_path`, 'stacks.base_path', 'stacks.included_paths' and 'stacks.excluded_paths' are correctly set in CLI config "+
+			"files or ENV vars.", j)
+		return fmt.Errorf(errorMessage)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to fmt.Errorf

(govet)